docs/documentation/configuration.mdx

```diff
@@ -0,0 +1,27 @@
+---
+title: Configuration
+---
+
+ParadeDB exposes a few Grand Unified Configuration (GUC) parameters that be turned
```

Comment:
We should link to Postgres GUCs here like you do for custom scans below, Elastic users will be confused/don't know what this is.